PhysicsTools/PatAlgos/python/slimming/MicroEventContent_cff.py

```diff
@@ -146,6 +146,13 @@
 cms.untracked.PSet(branch = cms.untracked.string("EcalRecHitsSorted_reducedEgamma_reducedESRecHits_*"),splitLevel=cms.untracked.int32(99)),
 ])
 
+_run2_HCAL_2018_extraCommands = ["keep *_packedPFCandidates_hcalDepthEnergyFractions_*"]
+from Configuration.Eras.Modifier_run2_HCAL_2018_cff import run2_HCAL_2018
+run2_HCAL_2018.toModify(MicroEventContentMC, outputCommands = MicroEventContentMC.outputCommands + _run2_HCAL_2018_extraCommands)
```

Comment:
is this really needed only for MC?
For data please modify ```MicroEventContent``` after L84.
I didn't realize this until I tested on data.

Reply:
Oops. Thanks. Needs to be fixed. Hold on.